packages/app_ui/app_gallery/lib/screens/widgets_screen.dart

```diff
@@ -0,0 +1,31 @@
+import 'package:app_gallery/widgets/widgets.dart';
+import 'package:flutter/material.dart';
+
+import 'widget_screens/widget_screens.dart';
+
+class WidgetsScreen extends StatefulWidget {
+  const WidgetsScreen({Key? key}) : super(key: key);
+
+  @override
+  State<WidgetsScreen> createState() => _WidgetsScreenState();
+}
+
+class _WidgetsScreenState extends State<WidgetsScreen> {
+  int currentIndex = 0;
+
```

Comment:
Is it used anywhere in the widget? If not, can be removed. Also the widget could became stateless then.